player.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, user: User):
         self.is_controller = False
 
         self.hand_of_cards: list[Any] = []
+        print("Hand od cards:", self.hand_of_cards)
```

Comment:
i assume this print was used for debugging, thus, should be removed.

Reply:
Yeah. Removed successfully